requirements.txt

```diff
@@ -5,6 +5,10 @@ duckdb==1.1.3
 fsspec>=2024.2.0
 ibis-framework[duckdb]==9.5.0
 mypy==1.8.0
+numpy<2.0.0
```

Comment:
it looks like there’s a specific version of numpy on line 11/15 below, is this duplicative?

Reply:
yes, removing this one. thx